### PR TITLE
Bhavya Singh assignment

### PR DESCRIPTION
**Pull request by bhavyasinghh**
this is the first git assignment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# github-assignment
```
Comment (rwest) [inline README.md]: I think you deleted this line by accident. Please could you add it back in a subsequent commit? thanks

Comment (bhavyasinghh) [reply README.md]: Dr. West, where am I supposed to add that?

Comment (bhavyasinghh) [reply README.md]: Alright! Done!

**Comment by bhavyasinghh** [comment]
Sorry about so many commits. Fixed the hashtag and I figured out why my avatar wasn't showing as well. Please let me know if there are any more problems!

**Comment by rwest** [comment]
Well done!

**Pull request merged**